src/software/geom/BUILD

```diff
@@ -218,6 +219,31 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "stadium",
+    srcs = ["stadium.cpp"],
+    hdrs = ["stadium.h"],
+    deps = [
+      ":convex_shape",
+      ":point",
+      ":segment",
+      ":rectangle",
+      ":circle",
+      ":polygon",
+    ],
+)
```

Comment:
nit: do we need all of these dependencies?
Can we remove `rectangle`, `circle`, and `convex_shape`?

Reply:
ConvexShape is needed as stadium extends it